### ISSUE TITLE
Why did I get a bad result?

### ISSUE THREAD

**Issue by konioyxgq** [eragonruan/refinenet#18]
I use mulit_gpu_train.py training data (the Tfrecord file you provided). Parameter ‘restore’ I chose False .When I choose true, the error is ‘'''NotFoundError (see above for traceback): Key feature_fusion/Conv_37/BatchNorm/beta/ExponentialMovingAverage not found in checkpoint'''’. so I chose False. And parameter ‘max_steps’ I chose 100000.
Training is done and the demo is running poorly.
why?

**Comment by konioyxgq** [comment]
@eragonruan
